CONTRIBUTING.md

```diff
@@ -169,6 +169,8 @@ Here is a list of the npm packages in this repository:
  | [webpack-dev-server](./npm/webpack-dev-server)     | `@cypress/webpack-dev-server`    | Webpack powered dev server for Component Testing.                |
  | [vue](./npm/vue)                                       | `@cypress/vue`                     | Cypress component testing for Vue 3.               |
  | [vue2](./npm/vue2)                                       | `@cypress/vue2`                     | Cypress component testing for Vue 2.               |
+ | [xpath](./npm/xpath)                                     | `@cypress/xpath`                    | Adds XPath command to Cypress.io test runner        |
+ | [grep](./npm/grep)                                       | `@cypress/grep`                     | Filter tests using substring                        |
```

Comment:
nit: abc order

Reply:
@jordanpowell88 can you also add the Schematic to this list? I noticed it's missing too.

Reply:
@admah Done in fcfa6c31d7de842a0725bf4bc521c8956464bcbd